csiplugin/connectors/rest_v2.go

```diff
@@ -137,21 +136,29 @@ func NewSpectrumRestV2(scaleConfig settings.Clusters) (SpectrumScaleConnector, e
 		glog.V(4).Infof("Created Spectrum Scale connector with SSL mode for %v", guiHost)
 	} else {
 		tr = &http.Transport{TLSClientConfig: &tls.Config{InsecureSkipVerify: true}}
-                glog.V(4).Infof("Created Spectrum Scale connector without SSL mode for %v", guiHost)
+		glog.V(4).Infof("Created Spectrum Scale connector without SSL mode for %v", guiHost)
 	}
 
-	return &spectrumRestV2{httpClient: &http.Client{Transport: tr}, endpoint: endpoint, user: guiUser, password: guiPwd}, nil
+	return &spectrumRestV2{
+		httpClient: &http.Client{
+			Transport: tr,
+			Timeout:   time.Second * 10,
```

Comment:
REST API does not have standard timeout setting. 10 Sec seems reasonable but on heavy loaded system we should run some test to verify. Also I see lots of format changes in this checkin along with timeout changes.

Reply:
Hey Deepak, the original commit was before the gofmt changes were in devel, so that is why it was all funky.  I merged up to the latest devel now, so there shouldn't be any format changes.

For the timeout, yeah I think 10 seconds is reasonable.  I think if 10 seconds is not enough, we should report error to CSI correctly, and then it will retry with backoff.  Still, I agree it should be tested under load